requirements.txt

```diff
@@ -5,6 +5,7 @@ colorlog
 cssselect
 ipaddress; python_version < '3.0'
 lxml>=3.8.0
+mock; python_version < '3.3'
```

Comment:
Installing mock framework for all the users is undesired, because it is unit-testing dependency. You'll have to remove it from requirements.txt and add into appveyor and travis configs.

Reply:
@undera fixed in https://github.com/Blazemeter/taurus/pull/697/commits/b7d2ed941f5425a72873f47c8bcf817bb9e61193

Thanks for mentioning this one. I noticed things like `pytest` and `nose` in requirements.txt, so I was unsure where to place it, but I agree with your reasoning on it.